content/docs/source-git/work-with-source-git/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Working with source-git"
+date: 2020-06-24T23:59:59+01:00
+draft: false
+weight: 3
+bookCollapseSection: true
+---
+
+# Working with source-git
+
+This document is focused on how packit treats a source-git repo.
+
+If you are looking for the design concept of source-git, please head on here:
+[docs/source-git-design]({{< ref "/docs/source-git/design.md" >}})
+
+The source-git concept is fairly simple - take an upstream repo and
+shove downstream content on top of it. Easy right?
+
+Packit is then able to work with such a repo and:
+
+* Create SRPMs
+
+* Sync the state into dist-git
+
+* And you are able to utilize all features of packit-service: RPM builds of PRs,
+  tests on those builds, and getting feedback in pull (merge) requests.
+
+Let's describe all of these in detail.
+
+
+## Upstream git history or just an unpacked tarball?
+
+Your source-git repo can contain upstream git history if you want. Or it
+doesn't. The alternative is to unpack a tarball corresponding to an upstream
```

Comment:
Wording feels sloppy, could you make it a little bit more concrete and structured?

Would it make sense to reference docs for creating such source-git repositories?

Reply:
I am going to drop that section for good, you're right that it doesn't feel right to have it here.